src/main/docs/guide/graalvm.adoc

```diff
@@ -0,0 +1,17 @@
+
+Micronaut Elasticsearch is compatible with https://www.graalvm.org/[GraalVM] out of the box. The only thing needed is
+to add some dependencies to make logging work.
+
+[source,groovy]
+----
+// Enable logging on JVM
+runtimeOnly 'org.slf4j:log4j-over-slf4j:1.7.30'
+
+// Additional dependencies for logging on GraalVM
+implementation 'org.apache.logging.log4j:log4j-api:2.13.3'
+implementation 'org.apache.logging.log4j:log4j-core:2.13.3'
+----
```

Comment:
Could you use `dependency:org.apache.logging.log4j:log4j-api[version="2.13.3"]` for these snippets to support Maven too

Reply:
done